src/parser.h

```diff
@@ -21,6 +21,8 @@ namespace ada {
     bool password_token_seen = false;
 
     const char* pointer;
+    const char* pointer_start;
+    const char* pointer_end;
```

Comment:
I don’t think it’s necessarily guaranteed that `std::string_view::iterator` is the same as `const char*`?

Reply:
You're right. Updated the pull request. Thanks!